gensim/models/hdpmodel.py

```diff
@@ -34,11 +34,11 @@
 from __future__ import with_statement
 
 import logging, time
-import numpy as np
```

Comment:
Yeah, I'll make it back to `np`. 
But I still think it should be uniform throughout the package - will open another PR for the other 3.